examples/js/loaders/GLTF2Loader.js

```diff
@@ -385,6 +392,24 @@ THREE.GLTF2Loader = ( function () {
 
 	}
 
+	/* DRACO MESH COMPRESSION EXTENSION */
+
+	function GLTFDracoMeshCompressionExtension () {
+
+		this.name = EXTENSIONS.KHR_DRACO_MESH_COMPRESSION;
+
+		this.dracoLoader = new THREE.DRACOLoader();
+
+	}
+
+	GLTFDracoMeshCompressionExtension.prototype.decodePrimitive = function ( primitive, dependencies ) {
+
+		var bufferViewID = primitive.extensions[ this.name ].bufferView;
```

Comment:
@donmccurdy Thanks for the remainder! The version check should be available in our latest javascripts posted in our master branch. You can check whether the provided version is supported using function `DracoModule.isVersionSupported(versionString)`

The only problem is that the function needs to be called from and instance of `DracoModule` which is in your case automatically instantiated inside the class `THREE.DRACOLoader` and as far as I can tell, there is currently no easy way get the module out of the loader, so we may have to either add the version check functionality to `THREE.DRACOLoader` or we can make the `DracoModule` accessible to other classes. Alternatively, you can just create a new instance of `DracoModule` in your code, but that would be an overkill for a simple version check (it's actually quite costly to instantiate it).